rust_src/src/lisp.rs

```diff
@@ -339,6 +339,17 @@ impl LispObject {
         }
     }
 
+    #[inline]
+    pub fn as_fixnum_coerce_marker_or_error(self) -> EmacsInt {
```

Comment:
I think it's slightly unfortunate that the other `.as_foo` methods return Rust wrappers, whereas `.as_fixnum_coerce_marker_or_error` just returns an `EmacsInt`. I don't have any better ideas though.